faithcomesbyhearing/dbp
# FCBH-1174 Highlights issues

# Description
- Added `color` filter to `Highlights`GET  endpoint
- Added `highlights/colors` GET endpoint in order to get all the colors that the user has on the highlights

## Issue Link
Story: [FCBH-1174](https://fullstacklabs.atlassian.net/browse/FCBH-1174) 

## How Do I QA This
- Run `https://dbp.test/open-api-v4.json` to get the new documentation
- Test `http://dbp.test/api/users/highlights/colors?v=4&key={KEY}&api_token={API_TOKEN}` `GET` endpoint and verify that you get the list of colors that the user has
- Test `http://dbp.test/api/users/{USER_ID}/highlights?v=4&key={KEY}&color=addd79` `GET` endpoint and verify that the endpoint is being filtered by the color
## Checklist
